.github/ISSUE_TEMPLATE/feature.yaml

```
@@ -0,0 +1,31 @@
+name: Feature Request
+title: '[FEATURE]: '
+labels: ['enhancement']
+description: |
+  Please use this template to submit a feature request for ngx-datatable. 
+  Ensure that you have searched the existing issues to avoid duplicates.
```

Same as above:

```suggestion
  Ensure that you have searched for existing issues to avoid duplicates.
```

or

```suggestion
  Ensure that you have searched trough the existing issues to avoid duplicates.
```